pkg/lvgl/Makefile

```diff
@@ -0,0 +1,24 @@
+PKG_NAME=lvgl
+PKG_URL=https://github.com/littlevgl/lvgl
+PKG_VERSION=v6.1.1
```

Comment:
This should be a commit hash with the version as a comment

Reply:
IMHO, that doesn't make any difference